package.json

```
@@ -58,38 +61,36 @@
   },
   "dependencies": {
     "@conda-store/conda-store-ui": "2023.10.2",
-    "@jupyterlab/application": "^3.1.0",
-    "@jupyterlab/apputils": "^3.1.0",
-    "@jupyterlab/mainmenu": "^3.1.0",
-    "@jupyterlab/settingregistry": "^3.1.0",
-    "@jupyterlab/ui-components": "^3.1.0",
-    "@lumino/widgets": "^1.37.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "@jupyterlab/application": "^4.0.9",
+    "@jupyterlab/apputils": "^4.1.2",
+    "@jupyterlab/mainmenu": "^4.0.9",
+    "@jupyterlab/settingregistry": "^4.0.9",
+    "@jupyterlab/ui-components": "^4.0.9",
+    "@lumino/widgets": "^2.3.1"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@jupyterlab/builder": "^3.0.0 || ^4.0.0",
     "@jupyterlab/testutils": "^3.0.0 || ^4.0.0",
```

I would suggest dropping `^3.0.0 ||` part here:

```suggestion
    "@jupyterlab/builder": "^4.0.0",
    "@jupyterlab/testutils": "^4.0.0",
```